android/app/src/main/java/app/organicmaps/editor/ProfileFragment.java

```diff
@@ -61,14 +68,25 @@ private void refreshViews()
       ThreadPool.getWorker().execute(() -> {
         if (mEditsSent.getText().equals(""))
         {
-          UiUtils.hide(mEditsSent);
-          UiUtils.show(mEditsSentProgress);
+          UiUtils.show(mProfileInfoLoading);
+          UiUtils.hide(mUserInfoBlock);
         }
-        final int count = OsmOAuth.getOsmChangesetsCount(requireContext(), getParentFragmentManager());
+        final int profileEditCount = OsmOAuth.getOsmChangesetsCount(requireContext(), getParentFragmentManager());
+        final String profileUsername = OsmOAuth.getUsername(requireContext());
+        final Bitmap profilePicture = OsmOAuth.getProfilePicture(requireContext());
+
         UiThread.run(() -> {
-          mEditsSent.setText(String.valueOf(count));
-          UiUtils.show(mEditsSent);
-          UiUtils.hide(mEditsSentProgress);
+          mEditsSent.setText(NumberFormat.getInstance().format(profileEditCount));
+          mProfileName.setText(profileUsername);
+
+          // use generic img if user has no pfp or couldn't load
```

Comment:
```suggestion
          // Use generic image if user has no profile picture or it failed to load.
```

Is image loading repeated somehow in the case of some recoverable errors?

Reply:
it should be 👍️